paymentsheet/src/androidTest/java/com/stripe/android/paymentsheet/FlowControllerTest.kt

```diff
@@ -67,6 +68,13 @@ internal class FlowControllerTest {
             response.testBodyFromFile("payment-intent-confirm.json")
         }
 
+        networkRule.enqueue(
+            method("GET"),
```

Comment:
We should add a path or a host or something here so we know what it's referring to.

(host might not be super easy due to the way networkRule works, but we can still make it work pretty easily, happy to pair on it if that's the route you choose).

Reply:
`q.stripe.com` has no path. Everything is sent as query params 🙈

Reply:
I'll follow up with a change to make this have a host!